views/random-beer.hbs

```diff
@@ -0,0 +1,12 @@
+<div class:"row">
+
+    {{#each pickBeer}}
+
+    <div class="col-6 card object-fit-contain beer-card">
+        {{>beerpartial}}
+        <p class="beer-text">Food Pairing: {{food_pairing}}</p>
```

Comment:
en este párafro puedes usar un li para poner el array de food_pairing en lista:

`<p class="beer-text">Food Pairing: {{#each food_pairing}} <li>{{this}}</li> {{/each}}</p>`